### ISSUE TITLE
Form Label Accessibility

### ISSUE THREAD

**Issue by magalj** [HappyHobby/HappyHobby#56]
Form Label Accessibility
Minor
Description:
If a form control does not have a properly associated text label, the function or purpose of that form control may not be presented to screen reader users. Form labels also provide visible descriptions and larger clickable targets for form controls.
Reproduction:
1)	Developer view on home page / sign in (Used Chrome)
2)	Select email and password elements
3)	E-mail and Password<label for="input element name"> </label> in <form></form> not present
